providers/cloudns/api.go

```diff
@@ -107,6 +93,21 @@ func (c *cloudnsProvider) fetchAvailableNameservers() error {
 	return nil
 }
 
+func (c *cloudnsProvider) fetchAvailableTTLValues(domain string) error {
+	allowedTTLValues = nil
```

Comment:
Is `fetchAvailableTTLValues` going to make this API call more than once?  Would it be sufficient to call it only if needed?